components/gc-follow-us/_base.scss

```diff
@@ -310,7 +314,7 @@ $social-media-icons-size: 38px;
 	}
 
 	&.twitter {
```

Comment:
Note: this style is used here: https://wet-boew.github.io/GCWeb/templates/widgets-en.html

May be we should move that old working example page within this component instead.